src/api-service/__app__/onefuzzlib/azure/creds.py

```diff
@@ -23,24 +22,16 @@
 
 @cached
 def get_msi() -> MSIAuthentication:
+    allow_more_workers()
```

Comment:
Why do we do this here, too?

Reply:
If we have some shared setup we want to invoke in multiple places, maybe it should live in a single function, with some documentation explaining when we must invoke it, and why.

Reply:
It's our ugly hot-patching.  We're down to only a single instance of MSIAuthentication which will be going away soon.